web/src/components/ZWebsiteNav/ZWebsiteNav.vue

```diff
@@ -214,9 +229,25 @@ html:not(.no-script)
     .z-website-nav__button--mobile
       transform scale(1)
 
+.z-website-nav__separator
+  height size(4)
+  width size(.2)
+  background: $colors.neutral.dark-3
+  border-radius size(.5)
+
+.z-website-nav__logo__content
+  display flex
+  align-items center
+
 .z-website-nav__logo__wrapper, .z-website-nav__logo
   z-index
 
+.z-website-nav__logo
+  margin size(1)
+.z-website-nav
+  .z-website-nav__logo--blog
+    font-size size(3)
+
```

Comment:
use class `.z-website-nav__breadcrumb` instead
```suggestion
  .z-website-nav__breadcrumb
    font-size size(3)

```